tools/make/kube.mk

```diff
@@ -101,4 +103,20 @@ delete-cluster: $(tools/kind) ## Delete kind cluster.
 
 .PHONY: release-manifests
 release-manifests: $(tools/kustomize) ## Generate Kubernetes release manifests.
-	tools/hack/release-manifests.sh $(GATEWAY_API_VERSION) $(TAG)
+	@mkdir -p release-artifacts/
+	@curl -sLo release-artifacts/gatewayapi-crds.yaml ${GATEWAY_RELEASE_URL}
+	@echo "Added: release-artifacts/gatewayapi-crds.yaml"
+
+	@mkdir -pv $(OUTPUT_DIR)/manifests/provider
+	@cp -r $(KUBE_PROVIDER_DIR) $(OUTPUT_DIR)/manifests/provider
+	@mkdir -pv $(OUTPUT_DIR)/manifests/infra
+	@cp -r $(KUBE_INFRA_DIR) $(OUTPUT_DIR)/manifests/infra
+	@cd $(OUTPUT_DIR)/manifests/provider/config/envoy-gateway && $(ROOT_DIR)/$(tools/kustomize) edit set image envoyproxy/gateway-dev=$(IMAGE):$(TAG)
+	@$(tools/kustomize) build $(OUTPUT_DIR)/manifests/provider/config/default > release-artifacts/envoy-gateway.yaml
+	@$(tools/kustomize) build $(OUTPUT_DIR)/manifests/infra/config/rbac > release-artifacts/infra-manager-rbac.yaml
+	@$(tools/kustomize) build > release-artifacts/install.yaml
+	@echo "Generated: release-artifacts/install.yaml"
+
+	@cp examples/kubernetes/quickstart.yaml release-artifacts/quickstart.yaml
```

Comment:
lets rm `@` from all non echo commands to improve debuggability

Reply:
Okay